readme.md

```diff
@@ -0,0 +1,63 @@
+### WordCamp.org
+
+This repository mirrors https://meta.svn.wordpress.org/sites/trunk/wordcamp.org/ and can be used to contribute used Git instead of SVN.
+
+###### Setup 
+
+There are two primary ways to setup this repo for local development.
+
+1. Using VVV. WordCamp.org is available as part of WordPress.org's meta network, and can be setup by following meta-environment setup guide: https://github.com/WordPress/meta-environment/blob/master/docs/install.md
+
+1. If you'd like to only setup WordCamp.org and would like to use Docker, you can use inbuilt docker provisioning. Please follow these steps
+
+    **Note:** This will create `.data` directory which will contain MySQL files to persist data across docker restarts.
+
+    1. Make sure you have Docker installed and Docker daemon running on your system.
+    
+    1. Clone the repo: 
+        ```
+        git clone git@github.com:WordPress/wordcamp.org.git
+        ```
+    
+    1. Change into the directory, and run docker compose command. This could take some time depending upon the internet speed.
+        ```
+        cd wordcamp.org && docker-compose up --build
+        ```
+         This will also start the docker environment for WordCamp.org development.
+         
+    1. Add the following resolutions to your host file:
+        ```
+        127.0.0.1	central.wordcamp.test
+        127.0.0.1	wordcamp.test
+        127.0.0.1	2014.content.wordcamp.test
+        127.0.0.1	2014.misc.wordcamp.test
+        127.0.0.1	2014.new-site.wordcamp.test
+        ```
+        
+        `/wp-admin` pages for these sites should now be accessible. Use `admin` as username and `password` as password to login.
+        
+        **Note:** `https` URL scheme must be used to visit these sites. Security exception will be required in first time run.
+        
+    1. (Optional) Basic installation comes without themes and with only very basic plugins. Add plugins and themes which are installed in wordcamp.org if needed like so:
```

Comment:
`MUST` - Can this be automated as part of installation?

Reply:
Unfortunately I was not able to find a non complicated solution to do this. Main reason being that we mount the `wp-content` directory from host filesystem, which is going to overwrite Docker image's `wp-content` folder.

Reply:
🤔 I'm confused, I would think they'd be the same folder, so any changes within the container would be synced to the host. Is that not the case?

Reply:
Yup, so during the first time docker build process, a container image will be build. But while starting up the container, we will mount the host's `wp-content` directory inside that image (overwriting image's wp-content folder) so that files are synced. That's why if during the build process we install the plugins and themes, they will get overwritten. 

We can solve this by not mounting `wp-content` but instead sub-folders one by one, but that will make the config very big and hard to keep updated. Let me also take a look to see if there are other solutions, but so far this looks like to be the case.

Reply:
🤔 

huh. Maybe this is another case where we'd want [a script that the user runs on the host after installation](https://github.com/WordPress/wordcamp.org/pull/113#discussion_r284009060)?

Then it seems like we couldn't need to install anything during the initial build.

Reply:
We already have a script for installing plugins and themes in `install-pluign-theme.sh`. Perhaps we can mark this step as mandatory instead of optional. What do you think?

Reply:
That sounds good to me, since they won't really have a completely functioning environment until they do that.

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/113/commits/8e679936b8e51e534ca63ea91ea26639384690de